GOVERNANCE.md

```diff
@@ -0,0 +1,159 @@
+- Most decisions start by seeking Lazy Consensus <https://communitymgt.wikia.com/wiki/Lazy_consensus>.
+- If an objection is raised through the Lazy Consensus process, Deciders work together to seek an agreeable solution.
+- If Consensus can not be reached, but a decision must be made, the next step is try to attempt to agree that a vote should be called.
+  This is important, as it gives dissenting views a chance to request more information or raise further points.
+  If Deciders are the Oversight Committee, part of that responsibility is the final point of escalation, so agreeing to a vote is assumed if timeline doesn't allow the consensus process to continue.
+- If Deciders are Repository Maintainers, and they can't agree on calling a vote, they may escalate to the Oversight Committee.
+  This should only be done at this stage if:
+  1. An unmovable deadline is threatened by continuing the Consensus process; or
+  2. A Decider feels there is unreasonable blocking of both reaching Consensus and agreeing to a vote.
+      This should be rare, due to the social cost of discontinuing the Consensus process for this reason.
+      Most decisions should wait for the above process to take its course.
+- If Deciders agree to a vote, the default is a Simple Majority.
+- However, there are cases that require stronger voting – Supermajority or Unanimity – specified below:
+
+### Simple Majority Decisions
+
+If a vote is called, the default is a Simple Majority Vote <https://en.wikipedia.org/wiki/Majority>.
+
+### Supermajority Decisions
+
+If a vote is called, the following decisions require a Supermajority Vote <https://en.wikipedia.org/wiki/Supermajority>.
+
+- Oversight Committee: Enforcing a Code of Conduct violation by a community member.
+- Oversight Committee: Licensing and intellectual property changes.
+- Oversight Committee: Material changes to the Governance document.
+  - Note: editorial changes to governance may be made by lazy consensus, unless challenged.
+    These are changes which fix spelling or grammar, update work affiliation or similar, update style or reflect an outside and obvious reality.
+    They do not change the intention or meaning of anything in this document.
+
+### Unanimity Decisions
+
+If a vote is called, the following decisison require Unanimity <https://en.wikipedia.org/wiki/Unanimity>.
+
+- Repository Maintainers: Electing new Maintainers of the same repository.
+- Oversight Committee: Electing new Committee members.
+- Oversight Committee: Removing a Repository Maintainer or Committee member for any reason other than inactivity.
+
+## Proposal Process
+
+- Code changes should go through the pull request process, where the idea and implementation details can be publicly discussed with Maintainers, other contributors, and end users.
+  Pull requests should only be merged after receiving GitHub approval from at least one other Maintainer.
```

Comment:
> after receiving GitHub approval from at least one other Maintainer

If you are not a maintainer, what is the rule?

Reply:
You're right, the implication is at least one maintainer who is not the PR author. I will make this explicit.